product-service/src/types/api-types.d.ts

```diff
@@ -1,18 +1,43 @@
-export type IProduct = {
+export type Product = {
   id: string;
   title: string;
+  description: string;
   image: string;
   price: number;
 };
 
-export type IProductsData = {
-  products: IProduct[];
+export type Stock = {
```

Comment:
**Suggestion**: StockDTO, ProductDTO and these type should be store near dbClient

Reply:
ye it'd be better in a bigger project, I just made it easier for now as a trade-off for my convenience + it's just default place for storing types for serverless-auto-swagger plugin (which I found as bad idea to use already) so if you don't ming could I keep as is